tests/amazon/aws/hooks/test_sagemaker.py

```diff
@@ -0,0 +1,119 @@
+from unittest import mock
+
+import pytest
+from botocore.exceptions import ClientError
+
+from astronomer.providers.amazon.aws.hooks.sagemaker import SageMakerHookAsync
+
+
+class TestSagemakerHookAsync:
+    @pytest.mark.asyncio
+    @mock.patch("astronomer.providers.amazon.aws.hooks.sagemaker.SageMakerHookAsync.get_client_async")
+    async def test_sagemaker_processing_job(self, mock_client):
+        """Test sagemaker async hook function to describe the processing job using Aiobotocore lib"""
+        # mocking async context function with return_value of __aenter__
```

Comment:
```suggestion
```

Reply:
remove this comment line in other test cases also.

Reply:
Fixed